src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,69 @@
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  setSelectedTodo: (todo: null) => void,
```

Comment:
The type of `setSelectedTodo` parameter should be `Todo | null` instead of just `null`. Update the type to `(todo: Todo | null) => void`.

Reply:
Fix Vadim's comment